src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -91,10 +91,12 @@ public long BytesConsumed
         /// within the given UTF-8 encoded input text, skipping any white space.
         /// </summary>
         /// <remarks>
-        /// For JSON strings (including property names), this points to before the start quote.
-        /// </remarks>
-        /// <remarks>
-        /// For comments, this points to before the first comment delimiter (i.e. '/').
+        ///   <para>
+        ///     For JSON strings (including property names), this points to before the start quote.
+        ///   </para>
+        ///   <para>
+        ///     For comments, this points to before the first comment delimiter (i.e. '/').
+        ///   </para>
```

Comment:
These don't need to be separate paragraphs.  I think that just adds clutter when they could reasonably just be two sentences, one after the other.

Reply:
Fair. I think we have quite a lot of these two-paragraph remarks that doesn't necessarily have to be separate.

Should I merge all of them into one?